newswires/client/.eslintrc.cjs

```diff
@@ -39,6 +39,7 @@ module.exports = {
 				caughtErrorsIgnorePattern: '^_',
 			},
 		],
+		"@typescript-eslint/switch-exhaustiveness-check": "error",
```

Comment:
I added this because I'd quite like it, and it doesn't introduce any new errors, so it feels like a pretty reasonable thing to enforce. But I'm v open to other perspectives if people think there are situations where we wouldn't want to enforce exhaustiveness of `switch` cases

Reply:
sounds good to me, I'd expect wanting exhaustiveness checking to be a good default, and it's easy to opt out with an eslint-disable directive, or adding a `default` case